fractal_tasks_core/lib_regions_of_interest.py

```diff
@@ -28,14 +28,17 @@
 
 
 def prepare_FOV_ROI_table(
-    df: pd.DataFrame, metadata: list[str] = ["time"]
+    df: pd.DataFrame, metadata: tuple[str, ...] = ("time",)
 ) -> ad.AnnData:
     """
-    TBD
+    Prepare an AnnData table for fields-of-view ROIs.
 
     Args:
-        df: TBD
-        metadata: TBD
+        df:
+            Input dataframe, possibly prepared through
+            `parse_yokogawa_metadata`.
+        metadata:
```

Comment:
What do we pass as metadata? I couldn't find an example of this being used. Was this the old idea to pass additional info to ROI tables like e.g. timepoint info that was in the Yokogawa metadata?

Reply:
>  I couldn't find an example of this being used.

The default includes `time`. Is `time` a column which comes up in some yokogawa metadata parsing?

Reply:
I think it's related to this: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/25#issuecomment-1235544131

The time metadata can be relevant for some downstream analysis. We added it last summer to the parsing output from the yokogawa parser. Max brought up that he would need that.
I don't think it's ever been used so far. But yes, it would be to get that metadata from the Yokogawa parser into the obs of the ROI tables.

Let's leave this there for the time being